### PR TITLE
[21769] Add contributing.md

### PR DESCRIPTION
**Pull request by catalinaalcala**
## Ticket
- ACCSEC-21769

## Description
- Added CONTRIBUTING.md section and file to README
- Added CODE_OF_CONDUCT.md file
- Added License section to README

## Commit message
- docs: add contributing.md

## Testing
- [ ] Added unit tests
- [ ] Ran unit tests successfully
- [x] Added documentation for public APIs and/or Wiki

### CODE REVIEW

**Comment by AuthyApps** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The Pull Request title does not follow the convention [00000] PR Title text</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_Linter">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Contributing to Twilio Verify Android
+## What are good ways to contribute?
+We welcome any kind of contributions. From filling issues, to fixing typos and bugs all the way to creating entirely new templates.
```
Comment (yafuquen) [inline CONTRIBUTING.md:3]: please change `to creating entirely new templates` to `to implementing improvements`

Comment (catalinaalcala) [reply CONTRIBUTING.md]: Done

**Comment by AuthyApps** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">The Pull Request title does not follow the convention [00000] PR Title text</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_Linter">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by yafuquen — APPROVED** [review]

**Review by SergioFierro — APPROVED** [review]

**Pull request merged**